style.css

```diff
@@ -0,0 +1,398 @@
+  line-height: 24px;
+}
+
+.project .project-text p {
+  line-height: 24px;
+}
+
+.other-projects .project-image {
+  width: 100%;
+}
+
+.other-projects .project-text {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+  padding-bottom: 0;
+}
+
+.first-project.project .project-text h3 {
+  color: #091e42;
+  margin-top: 1rem;
+}
+
+.first-project.project .project-text p {
+  color: #091e42;
+}
+
+.project-text .languages {
+  display: flex;
+  gap: 4%;
+}
+
+.languages li {
+  background-color: rgba(255, 255, 255, 0.24);
+  padding: 3% 4%;
+  font-size: 15px;
+  line-height: 20px;
+  font-weight: 500;
+}
+
+.first-project .languages li {
+  color: #091e42;
+  border: 1px solid #8993a4;
+}
+
+.about-me button {
+  align-self: flex-start;
+}
+
+.first-project .project-text button {
+  width: fit-content;
+}
+
+.project-text button:hover {
+  cursor: pointer;
+  color: #ff6b00;
+  background-color: #fff5e1;
+}
+
+/* ============ End Works Section ============== */
+
+/* ============ About Section ============== */
+
+.about {
+  background-color: #1c1a19;
+  margin-top: 3rem;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 5%;
+  background:
+    var(--hero-backgroung-color)
+    url('./images/about-me-illustration.svg') no-repeat right bottom;
+  min-height: 80vh;
+  padding-top: 30vw;
+}
+
+.about-me h2 {
+  font-size: 70px;
+  color: #ff6b00;
+  font-weight: 400;
+  letter-spacing: 0.37px;
+  line-height: 88px;
+}
+
+footer .underline {
+  background-color: #dfe1e6;
+}
+
+.about .underline {
+  background-color: #fff;
+  margin: 5px auto;
+  width: 90%;
+}
+
+.skills {
+  color: #fff;
+  font-size: 22px;
+  font-family: 'Inter', sans-serif;
+  display: flex;
+  flex-direction: column;
+  gap: 1.4rem;
+  align-items: center;
+  padding: 10rem 0;
+}
+
+.skills .title {
+  display: flex;
+  gap: 10px;
+  align-items: baseline;
+  justify-content: center;
+}
+
+.skills .title h4 {
+  font-size: 32px;
+  font-family: 'Roboto', sans-serif;
+  margin-bottom: 1.4rem;
+}
+
+.lang .shape {
+  transform: rotate(45deg);
+}
+
+.skill .shape {
+  border-radius: 50%;
+}
+
+.skills .title .shape {
+  height: 18px;
+  width: 18px;
+  background-color: #ff6b00;
+}
+
+.skills ul {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+}
+
+/* ============ End About Section ============== */
+
+/* ============ Contact Section ============== */
+
+.contact {
+  padding: 5%;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+}
+
+form input {
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 0.6rem;
+}
+
+form textarea {
+  padding: 3%;
+  color: #3c3a39;
+  background: #fbf8f7;
+  border: 1px solid #fbf8f7;
+  font-style: italic;
+}
+
+/* ============ End Contact Section ============== */
+
+/* ============ Footer Section ============== */
+
+footer {
+  margin: 2rem 0;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
```

Comment:
Great job! Just remember it's always nice to use relative values that are responsive as opposed to absolute ones.